src/PortToDocs/src/libraries/ToDocsPorter.cs

```diff
@@ -107,9 +111,74 @@ public void Start()
             }
 
             PortMissingComments();
-            DocsComments.Save();
+        }
+
+        public void SaveToDisk() => DocsComments.SaveToDisk();
+
+        // Prints a final summary of the execution findings.
+        public void PrintSummary()
+        {
             PrintUndocumentedAPIs();
-            PrintSummary();
```

Comment:
The two print methods were merged into one, PrintSummary, and only called when using the tool with the filesystem.